### PR TITLE
Resolve minor issues in Console and My Account

### PR DESCRIPTION
**Pull request by thivi**

### CODE REVIEW

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 1]
```diff
@@ -572,18 +572,22 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                     ".validations.empty")
                             }
                             validation={ (value: string) => {
-
+                                // eslint-disable-next-line no-debugger
+                                debugger
```
Comment (brionmario) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:576]:
```suggestion
```

**Review by brionmario — CHANGES_REQUESTED** [review]

File: modules/react-components/src/components/avatar/avatar.tsx [hunk 2]
```diff
@@ -201,6 +204,27 @@ export const Avatar: FunctionComponent<PropsWithChildren<AvatarPropsInterface>>
...
 
+    /**
+     * Check if `image` is a valid image URL.
+     */
+    useEffect(() => {
+        if (image) {
+            if (React.isValidElement(image)) {
+                setIsImageValidUrl(false);
+            } else {
+                FormValidation.imageUrl(image as string).then((response) => {
```
Comment (brionmario) [inline modules/react-components/src/components/avatar/avatar.tsx:215]: `joi` sends a XHR GET request to validate the image and this causes CORS issues[1]. Use the `isValidImageURL` util method [2].

[1] https://github.com/wso2/product-is/issues/9378
[2] https://github.com/wso2/identity-apps/blob/master/modules/core/src/utils/image-utils.ts#L53

**Review by brionmario — CHANGES_REQUESTED** [review]

File: modules/react-components/src/components/avatar/avatar.tsx (hunk 2 above)
Comment (thivi) [reply modules/react-components/src/components/avatar/avatar.tsx:215]: Fied by 3699ebd2

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 1 above)
Comment (thivi) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: Fixed by 904dc42b

File: apps/console/src/features/groups/components/edit-group/edit-group-users.tsx [hunk 3]
```diff
@@ -464,45 +474,45 @@ export const GroupUsersList: FunctionComponent<GroupUsersListProps> = (props: Gr
...
-                                            "emptyPlaceholder.title") }
-                                        subtitle={ [
-                                            t("adminPortal:components.roles.edit.users.list." +
-                                                "emptyPlaceholder.subtitles", { type: "group" })
-                                        ] }
-                                        action={
-                                            !isReadOnly && (
-                                                <PrimaryButton
-                                                    data-testid={ `${ testId }-users-list-empty-assign-users-
+                                    <EmphasizedSegment>
```
Comment (jeradrutnam) [inline apps/console/src/features/groups/components/edit-group/edit-group-users.tsx:485]: Previous indentation is correct

File: apps/console/src/features/users/pages/users.tsx [hunk 4]
```diff
@@ -21,19 +21,26 @@ import { CommonHelpers } from "@wso2is/core/helpers";
...
 import { addAlert } from "@wso2is/core/store";
 import { LocalStorageUtils } from "@wso2is/core/utils";
-import { Button, ListLayout, PageLayout, PrimaryButton } from "@wso2is/react-components";
+import {
+    Button,
+    EmptyPlaceholder,
+    ListLayout,
+    PageLayout,
+    PrimaryButton
+}from "@wso2is/react-components";
```
Comment (jeradrutnam) [inline apps/console/src/features/users/pages/users.tsx:30]:
```suggestion
} from "@wso2is/react-components";
```

File: apps/myaccount/src/components/linked-accounts/linked-accounts.tsx [hunk 5]
```diff
@@ -108,27 +108,12 @@ export const LinkedAccounts: FunctionComponent<LinkedAccountsProps> = (props: Li
...
-
-                    return;
-                }
-
                 onAlertFired({
-                    description: t(
+                    description: error?.response?.data?.description ?? t(
                         "userPortal:components.linkedAccounts.notifications.addAssociation.genericError.description"
-                    ),
+                    ) ,
```
Comment (jeradrutnam) [inline apps/myaccount/src/components/linked-accounts/linked-accounts.tsx:114]:
```suggestion
                    ),
```

File: apps/myaccount/src/components/linked-accounts/linked-accounts.tsx [hunk 6]
```diff
@@ -108,27 +108,12 @@ export const LinkedAccounts: FunctionComponent<LinkedAccountsProps> = (props: Li
...
                     level: AlertLevels.ERROR,
-                    message: t(
+                    message: error?.response?.data?.message ?? t(
```
Comment (jeradrutnam) [inline apps/myaccount/src/components/linked-accounts/linked-accounts.tsx:116]: Is this the backend error we are showing here ? If shall we map it to a front-end error

File: apps/console/src/features/groups/components/edit-group/edit-group-users.tsx (hunk 3 above)
Comment (thivi) [reply apps/console/src/features/groups/components/edit-group/edit-group-users.tsx]: Resolved by e636c15b

File: apps/myaccount/src/components/linked-accounts/linked-accounts.tsx (hunk 6 above)
Comment (thivi) [reply apps/myaccount/src/components/linked-accounts/linked-accounts.tsx]: Resolved by 97406f11

**Review by brionmario — APPROVED** [review]

**Pull request merged**